tests/rest/admin/test_room.py

```diff
@@ -24,9 +24,10 @@
 import synapse.rest.admin
 from synapse.api.constants import EventTypes, Membership, RoomTypes
 from synapse.api.errors import Codes
-from synapse.handlers.pagination import PaginationHandler, PurgeStatus
+from synapse.handlers.pagination import DeleteStatus, PaginationHandler, PurgeStatus
```

Comment:
Feels kinda like we don't need `PurgeStatus` given we track things in `DeleteStatus` and persist that to the database.